pytest_inmanta/parameters.py

```diff
@@ -142,6 +143,7 @@ def get_help(self) -> str:
 pip_index_url = ListTestParameter(
     argument="--pip-index-url",
     environment_variable="PIP_INDEX_URL",
+    legacy_environment_variable="INMANTA_PIP_INDEX_URL",
```

Comment:
Why? This never was an environment variable we accept, was it?

Reply:
it was (see the diff)

Reply:
Oh, indeed, sorry